interfaces/AST_generic_v1.atd

```diff
@@ -420,7 +420,7 @@ type concat_string_kind = [
    * and some semgrep users may want to explicitely match only f-strings,
    * which is why we record this information here.
   *)
-  | FString (* Python *)
+  | FString of string(* Python *)
```

Comment:
Are we allowed to modify this AST?

```
 * DO NOT MODIFY THIS FILE. The "API" defined here should be stable
 * across multiple versions of Semgrep. If you really want to change
 * things, you'll have to define an AST_generic_v2.atd
```

Reply:
@aryx could you confirm whether this is OK?

Reply:
Yes we're not supposed to modify this file because there is a _v1 so it's supposed to be the same forever.
Nobody use those (supposidely stable) exported ASTs so it's also fine.
But usually I don't modify and put some raise Todo in the AST_generic converter to AST_generic_v1